src/shared/ui/navigation/DropdownMenu/DropdownMenu.tsx

```diff
@@ -0,0 +1,201 @@
+        className,
+      )}
+      {...props}
+    />
+  );
+});
+
+export const DropdownMenuContent = React.forwardRef<
+  React.ElementRef<typeof DropdownMenuPrimitive.Content>,
+  React.ComponentPropsWithoutRef<typeof DropdownMenuPrimitive.Content>
+>(function DropdownMenuContent(
+  { className, align = "end", sideOffset = 8, ...props },
+  ref,
+) {
+  return (
+    <DropdownMenuPrimitive.Portal>
+      <DropdownMenuPrimitive.Content
+        ref={ref}
+        align={align}
+        sideOffset={sideOffset}
+        className={twJoin(
+          "z-50 min-w-[12.25rem] overflow-hidden rounded-border-radius-base bg-background-elevated-base p-spacing-tighter-4 text-color-base shadow-md",
+          "data-[state=open]:animate-in data-[state=closed]:animate-out data-[state=closed]:fade-out-0 data-[state=open]:fade-in-0 data-[state=closed]:zoom-out-95 data-[state=open]:zoom-in-95 data-[side=bottom]:slide-in-from-top-2 data-[side=left]:slide-in-from-right-2 data-[side=right]:slide-in-from-left-2 data-[side=top]:slide-in-from-bottom-2",
+          className,
+        )}
+        {...props}
+      />
+    </DropdownMenuPrimitive.Portal>
+  );
+});
+
+export const DropdownMenuItem = React.forwardRef<
+  React.ElementRef<typeof DropdownMenuPrimitive.Item>,
+  React.ComponentPropsWithoutRef<typeof DropdownMenuPrimitive.Item> & {
+    inset?: boolean;
+  }
+>(function DropdownMenuItem({ className, inset, ...props }, ref) {
+  return (
+    <DropdownMenuPrimitive.Item
+      ref={ref}
+      className={twJoin(
+        "relative flex cursor-default select-none items-center rounded-sm py-2.5 pl-spacing-tighter pr-spacing-tighter-2 text-font-size-smaller outline-none transition-colors focus:bg-background-tinted-highlight data-[disabled]:pointer-events-none data-[disabled]:opacity-50",
+        inset && "pl-8",
+        className,
+      )}
+      {...props}
+    />
+  );
+});
+
+export const DropdownMenuCheckboxItem = React.forwardRef<
+  React.ElementRef<typeof DropdownMenuPrimitive.CheckboxItem>,
+  React.ComponentPropsWithoutRef<typeof DropdownMenuPrimitive.CheckboxItem>
+>(function DropdownMenuCheckboxItem(
+  { className, children, checked, ...props },
+  ref,
+) {
+  return (
+    <DropdownMenuPrimitive.CheckboxItem
+      ref={ref}
+      className={twJoin(
+        "focus:bg-accent focus:text-accent-foreground relative flex cursor-default select-none items-center rounded-sm py-1.5 pl-8 pr-2 text-sm outline-none transition-colors data-[disabled]:pointer-events-none data-[disabled]:opacity-50",
+        className,
+      )}
+      checked={checked}
+      {...props}
+    >
+      <span className="absolute left-2 flex h-3.5 w-3.5 items-center justify-center">
+        <DropdownMenuPrimitive.ItemIndicator>
+          <IoMdCheckmark className="h-4 w-4" />
```

Comment:
Same comment about classes:

1. `focus:bg-accent focus:text-accent-foreground` do nothing I think
2. Can we use our `spacing-` variables instead of `py-1.5 pl-8 pr-2` `left-2 flex h-3.5 w-3.5` `h-4 w-4`
3. Can we use our `font-size-` variable instead of `text-sm`
4. For the same reasons as above, I'm not sure `transition-colors` is really useful here?

Reply:
`<DropdownMenuCheckboxItem>` is not yet used but will probably be for the dropdown used in the sidebar, so I've made the corrections !